src/action/setting.js

```diff
@@ -11,6 +12,14 @@ class SettingAction {
     }
     this._store.settings.unit = unit;
   }
+
+  setFiatCurrency({ fiat }) {
+    if (!FIATS[fiat]) {
+      throw new Error(`Invalid fiat currency: ${fiat}`);
+    }
+    this._store.settings.fiat = fiat;
+    this._wallet.getExchangeRate();
```

Comment:
Added fetching of exchange rate in https://github.com/lightninglabs/lightning-app/pull/409/commits/850ad99a5733387c129e0a85fa0821ea54ad5f89